app/client/cypress/e2e/Regression/ClientSide/Widgets/TableV2/Table_InfiniteScroll_spec.ts

```
@@ -48,12 +46,12 @@ describe(
       agHelper.AssertClassExists(locators._jsToggle("tabledata"), "is-active");
     });
 
-    it("1. should enable infinite scroll and verify records are loaded and loaded more records works", () => {
+    it.only("1. should enable infinite scroll and verify records are loaded automatically when scrolling", () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)